Immense.RemoteControl.Desktop.Shared/Services/DtoMessageHandler.cs

```diff
@@ -163,13 +153,23 @@ private async Task ClipboardTransfer(DtoWrapper wrapper)
         }
     }
 
+    private void CtrlAltDel()
+    {
+        if (OperatingSystem.IsWindows())
+        {
+            // Might as well try both.
+            User32.SendSAS(false);
```

Comment:
Would be nice to use Named args here, wasn't sure what the bool indicated
```suggestion
            User32.SendSAS(AsUser: false);
```